designer/server/src/common/components/service-header/template.njk

```diff
@@ -72,6 +74,8 @@ Component options:
             {% endif %}
           </span>
         </a>
+        {# Environment identification tag #}
```

Comment:
Super minor but this comment will inject a new line

Many many more haven't been tackled so not a blocker in the slightest

```suggestion
        {# Environment identification tag -#}
```